connector_woocommerce/components/exporter.py

```diff
@@ -0,0 +1,355 @@
+# © 2009 Tech-Receptives Solutions Pvt. Ltd.
+# © 2018 Serpent Consulting Services Pvt. Ltd.
```

Comment:
The copyright on several files have been stripped, as they are strongly similar than: https://github.com/OCA/connector-magento/blob/10.0/connector_magento/components/exporter.py